File: fibonacci.js
const readline = require('readline');

function fibonacci(n) {
  if (n <= 1) {
    return n;
  } else {
    return fibonacci(n - 1) + fibonacci(n - 2);
  }
}

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question("Digite um número: ", function(n) {
  const fib = fibonacci(parseInt(n));
  console.log(`O ${n} na Fibonacci é: ${fib}`);
  rl.close();
});

File: fibonacci.java
import java.util.Scanner;

public class fibonacci {
    public static int fibo(int n) {
        if (n == 0 || n == 1) {
            return n;
        } else {
            return fibo(n - 1) + fibo(n - 2);
        }
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite um número inteiro: ");
        int n = scanner.nextInt();

        int fibonacci = fibo(n);
        System.out.println("O número de Fibonacci correspondente é: " + fibonacci);

        scanner.close();
    }
}
